install_config/router/default_haproxy_router.adoc

```diff
@@ -1122,6 +1122,44 @@ $ oc set env dc/router -c router DROP_SYN_DURING_RESTART=true
 If you used a non-default name for the router, you must change *_dc/router_*
 accordingly.
 
+[[deploy-router--arp-cach-tuning-for-large-scale-clusters]]
+== ARP Cache Tuning for Large-scale Clusters
+
+In {product-title}} clusters with large numbers of routes (greater than the
+value of `net.ipv4.neigh.default.gc_thresh3`, which is `1024` by default), you
+must increase the default values of sysctl variables to allow more entries in
+the ARP cache.
+
+In clusters, the kernel messages would be similar to the following:
+
+----
+[ 1738.811139] net_ratelimit: 1045 callbacks suppressed
+[ 1743.823136] net_ratelimit: 293 callbacks suppressed
+----
+
+When using the `oc` client, the commands fail and the user receives the following:
+
+----
+Unable to connect to the server: dial tcp: lookup <hostname> on <ip>:<port>: write udp <ip>:<port>-><ip>:<port>: write: invalid argument
+----
+
+To verify the actual amount of ARP entries for IPv4, run the following:
+
+----
+# ip -4 neigh show nud all | wc -l
+----
+
+If the number begins to approach the `net.ipv4.neigh.default.gc_thresh3`
+threshold, increase the values. The following sysctl values are recommended for large-scale clusters:
```

Comment:
Added a sentence on running it on the router pods. There's a chance it won't be on the infra nodes, but if so, it'll be easier to find.